release-notes/8.0/known-issues.md

```diff
@@ -13,6 +13,41 @@ SDK Resolver Failure: "The SDK resolver "Microsoft.DotNet.MSBuildSdkResolver" fa
 File name: '...\sdk-manifests\8.0.100-rc.1\microsoft.net.workload.emscripten.current\WorkloadManifest.json'
 ```
 
+### [.NET SDK 8.0.1xx RC2] `dotnet tool restore` may fail with a file access exception when restoring multiple tools
+
+When running `dotnet tool restore` with a tools config which contains multiple .NET tools, the command can fail with an exception similar to the following:
+
+```
+Unhandled exception: System.AggregateException: One or more errors occurred. (The process cannot access the file 'C:\Users\Username\AppData\Local\Temp\a75617cf-9767-45ca-aaed-34da2edd223e\project.assets.json' because it is being used by another process.)
+ ---> System.IO.IOException: The process cannot access the file 'C:\Users\Username\AppData\Local\Temp\a75617cf-9767-45ca-aaed-34da2edd223e\project.assets.json' because it is being used by another process.
+   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
+   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
+   at System.IO.Strategies.OSFileStreamStrategy..ctor(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
+   at System.IO.Strategies.FileStreamHelpers.ChooseStrategyCore(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize, Nullable`1 unixCreateMode)
+   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
+   at NuGet.ProjectModel.LockFileFormat.Write(String filePath, LockFile lockFile)
+   at Microsoft.DotNet.Cli.ToolPackage.ToolPackageDownloader.CreateAssetFile(PackageId packageId, NuGetVersion version, DirectoryPath packagesRootPath, DirectoryPath assetFileDirectory, String runtimeJsonGraph, String targetFramework) in C:\git\dotnet-sdk-8\src\Cli\dotnet\ToolPackage\ToolPackageDownloader.cs:line 329
+   at Microsoft.DotNet.Cli.ToolPackage.ToolPackageDownloader.<>c__DisplayClass8_0.<InstallPackage>b__0() in C:\git\dotnet-sdk-8\src\Cli\dotnet\ToolPackage\ToolPackageDownloader.cs:line 130
```

Comment:
do we have any standard policy around normalizing paths in error text we include? The path is pretty generic but looks like a local git repo.

Reply:
Deterministic builds would result in cutting off the prefix portion of the path and replacing it with `/_/` - so we could do that here. Replace all of the `c:\git\dotnet-sdk-8` with `/_/` and call it good.

Alternatively for purposes of this text we could just remove the paths entirely.